doc/content/integrations/mqtt/Connecting to Node-RED/_index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Connecting to Node-RED"
+description: ""
+---
+
+[Node-RED](https://nodered.org/) is free, JavaScript-based development tool for visual programming, developed to ease the process of wiring together hardware devices, APIs and online services.
+
+The MQTT server that is exposed by the Application Server can be connected to Node-RED. This kind of integration allows setting up a Node-RED flow that listens to events and messages sent by the end devices. Also, it is possible to send messages to the end devices.
+
+This guide contains the instructions for setting up these flows, but first make sure to properly [install the Node-RED](https://nodered.org/docs/getting-started/local).
+
+## Setup
+
+First, you need to run the Node-RED by using the following command in terminal:
+```bash
+$ node-red 
+```
+```bash
+# Tip: Node-RED runs locally on port 1880 by default, so make sure to allow the service to run on this port with:
+
+$ sudo ufw allow 1880
+```
```

Comment:
Use empty lines between blocks. See also the documentation requirements: https://github.com/TheThingsNetwork/lorawan-stack/blob/develop/DEVELOPMENT.md#documentation-site